sys/linux/socket_netlink_route_sched.txt

```diff
@@ -231,6 +232,21 @@ fq_codel_policy [
 	TCA_FQ_CODEL_MEMORY_LIMIT	nlattr[TCA_FQ_CODEL_MEMORY_LIMIT, int32]
 ] [varlen]
 
+fq_pie_policy [
+	TCA_FQ_PIE_LIMIT		nlattr[TCA_FQ_PIE_LIMIT, int32]
+	TCA_FQ_PIE_FLOWS		nlattr[TCA_FQ_PIE_FLOWS, int32[1:65535]]
+	TCA_FQ_PIE_TARGET		nlattr[TCA_FQ_PIE_TARGET, int32]
+	TCA_FQ_PIE_TUPDATE		nlattr[TCA_FQ_PIE_TUPDATE, int32]
+	TCA_FQ_PIE_ALPHA		nlattr[TCA_FQ_PIE_ALPHA, int32[0:32]]
+	TCA_FQ_PIE_BETA			nlattr[TCA_FQ_PIE_BETA, int32[0:32]]
+	TCA_FQ_PIE_QUANTUM		nlattr[TCA_FQ_PIE_QUANTUM, int32]
+	TCA_FQ_PIE_MEMORY_LIMIT		nlattr[TCA_FQ_PIE_MEMORY_LIMIT, int32]
+	TCA_FQ_PIE_ECN_PROB		nlattr[TCA_FQ_PIE_ECN_PROB, int32[0:100]]
+	TCA_FQ_PIE_ECN			nlattr[TCA_FQ_PIE_ECN, int32[0:1]]
```

Comment:
We have a bit more handy alias `bool32` for `int32[0:1]`:
https://github.com/google/syzkaller/blob/3c7fef3361a6007112b26adc1c5a550189ef43fe/docs/syscall_descriptions_syntax.md#type-aliases
Please use it for TCA_FQ_PIE_ECN/TCA_FQ_PIE_BYTEMODE/TCA_FQ_PIE_DQ_RATE_ESTIMATOR.